src/App.js

```diff
@@ -1,14 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
+import CardUi from './components/CardUi'
+import NavBar from './components/Navbar';
 import './styles/App.scss';
+import restaurants from './data/restaurants.json';
+import ButtonCycleWrapper from './components/ButtonCycleWrapper';
 
 function App() {
+
+  let [cardCycle, setCardCycle] = useState(ShuffleArr(deepCopy(restaurants)));
+
+  const handleClick = (e) => {
+    const value = e.target.id;
+    if (value === 'skip') {
+      setCardCycle(cardCycle.splice(1, cardCycle.length));
```

Comment:
a big nono.
splice will mutate the original array (the state).
mutate the state only with setState.
solution: use slice instead. it will return a new array instead of mutating the original one

Reply:
ahhh true, don't know how I didn't pick up on this. I'll fix it right away

Reply:
fixed on https://github.com/chingu-voyages/v14-geckos-team-01/pull/5/commits/19eb4cf85c0d07aa48d903a1aa7112a794a90450